### PR TITLE
fix: resurrect pinpoint event submission test

### PR DESCRIPTION
**Pull request by jamesonwilliams**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-android-sdk-testutils/src/main/java/com/amazonaws/testutils/util/AirplaneMode.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+public final class AirplaneMode {
+    private static final int TIMEOUT_MS = 500;
+
+    private AirplaneMode() {}
+
+    public static void enable() {
+        setAirplaneMode(true);
+    }
+
+    public static  void disable() {
```
Comment (raphkim) [inline aws-android-sdk-testutils/src/main/java/com/amazonaws/testutils/util/AirplaneMode.java:37]: extra space between `static` and `void`

File: aws-android-sdk-testutils/src/main/java/com/amazonaws/testutils/util/AirplaneMode.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import android.content.Intent;
+import android.provider.Settings;
+import androidx.test.uiautomator.By;
+import androidx.test.uiautomator.BySelector;
+import androidx.test.uiautomator.UiDevice;
+import androidx.test.uiautomator.Until;
+
+import static androidx.test.platform.app.InstrumentationRegistry.getInstrumentation;
+
+public final class AirplaneMode {
```
Comment (raphkim) [inline aws-android-sdk-testutils/src/main/java/com/amazonaws/testutils/util/AirplaneMode.java:28]: very cool!

File: aws-android-sdk-testutils/src/main/java/com/amazonaws/testutils/util/InternetConnectivity.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.net.Socket;
+import java.util.concurrent.TimeUnit;
+
+public final class InternetConnectivity {
+    private InternetConnectivity() {}
+
+    public static void goOnline() {
```
Comment (raphkim) [inline aws-android-sdk-testutils/src/main/java/com/amazonaws/testutils/util/InternetConnectivity.java:26]: do we need to synchronize these methods? Can there be a race condition between checking for connection status vs changing connection status?

Comment (jamesonwilliams) [reply aws-android-sdk-testutils/src/main/java/com/amazonaws/testutils/util/InternetConnectivity.java:26]: It couldn't hurt. I'll add a big ol'
```java
synchronized (InternetConnectivity.class) {
}
```
around it.

File: aws-android-sdk-testutils/src/main/java/com/amazonaws/testutils/util/AirplaneMode.java (hunk 1 above)
Comment (jamesonwilliams) [reply aws-android-sdk-testutils/src/main/java/com/amazonaws/testutils/util/AirplaneMode.java:37]: Thanks! will remove.

**Pull request merged**